updating/understanding-openshift-updates.adoc

```diff
@@ -0,0 +1,15 @@
+:_content-type: ASSEMBLY
+[id="understanding-openshift-updates"]
+= Understanding {product-title} updates
+include::modules/common-attributes.adoc[]
+:context: understanding-openshift-updates
+
+toc::[]
+
+With {product-title} 4, you can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (`oc`). Platform administrators are automatically notified when an update is available for their cluster. Updates can be applied either from the `oc` command-line interface (CLI) or the web console.
+
+The OpenShift Update Service (OSUS) builds a graph of upgrade possibilities based on release images in the registry.
+The graph is based on supported, tested update paths from a specific version. {product-title} clusters connect to the Red Hat Hybrid Cloud servers and identify which clusters the user is running, along with the version information. OSUS responds with information about known update targets. Either a cluster administrator or an automatic update controller edits the custom resource (CR) of the Cluster Version Operator (CVO) with the new version for OSUS to update to.
```

Comment:
`candidate-4.y` channels include releases and updates that are not supported.  Can we say "...recommended, tested..." or some such, to be more generic?  And leave supported-ness up to the channel semantics?

Reply:
another nit: `with the new version for OSUS to update to.`  It's actually the CVO making the update.  The flow is:

1. CVO polls OSUS for possible targets.
2. CVO stores OSUS response in ClusterVersion.
3. Admin/bot selects a possible target from the ClusterVersion `status` (or brings in their own explicit pullspec, but probably not worth involving this path here), and updates ClusterVersion `spec.desiredUpdate`.
4. CVO sees the re-target request in `spec.desiredUpdate`, performs some "do I really want to do this?" validation, and then begins updating.

So OSUS is out of the picture well before step 4's "we're starting to update".